hbase-shell/src/main/ruby/shell/commands/clone_snapshot.rb

```diff
@@ -33,13 +33,17 @@ def help
 newly created table.
 
   hbase> clone_snapshot 'snapshotName', 'namespace:tableName', {RESTORE_ACL=>true}
+
+StoreFileTracker implementation used after restore can be set by the following command.
+  hbase> clone_snapshot 'snapshotName', 'namespace:tableName', {CLONE_SFT=>FILE}
```

Comment:
should be `{CLONE_SFT=>"FILE"}`?

Reply:
Based on other examples the sell prefers single quotes, but you are right those are missing.